build-bin/docker/docker_test_image

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright 2019-2020 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+set -ue
+
+# test a an image by awaiting its HEALTHCHECK
+
+docker_image=${1?docker_image is required, notably without a tag. Ex openzipkin/zipkin}
+
+# This just makes sure containers run and the HEALTHCHECK works (for now..)
```

Comment:
this script is revised based on experiences here, especially around ensuring you can log output of failed container when docker-compose is used. later, I will copy it across the other repos.